Sources/Internal/CoreStoreFetchRequest+CoreStore.swift

```diff
@@ -36,6 +36,6 @@ internal extension CoreStoreFetchRequest {
     @nonobjc
     internal func dynamicCast<U: NSFetchRequestResult>() -> NSFetchRequest<U> {
         
-        return unsafeBitCast(self, to: NSFetchRequest<U>.self)
+        return unsafeDowncast(self, to: NSFetchRequest<U>.self)
```

Comment:
Not really. CoreStoreFetchRequest was meant to be an internal type. 

`dynamicCast` was written to be able to cast to any `U` (within CoreStore of course). Come to think of it, I think unsafeBitCast and unsafeDowncast would behave the same here because `NSFetchRequestResult` is an Objective-C protocol (and thus `id` in memory). But I just think the semantics of "downcast" doesn't fit the "dynamicCast" expectations.